at_lookup/lib/src/at_lookup_impl.dart

```diff
@@ -267,7 +267,20 @@ class AtLookupImpl implements AtLookUp {
     }
     // If response starts with error:, throw AtLookupException.
     if (_isError(verbResult)) {
-      verbResult = verbResult.replaceAll('error:', '');
+      verbResult = verbResult.replaceFirst('error:', '');
+      // If condition to verify if verbResult is a JSON Encoded String.
+      if (verbResult.startsWith('{') && verbResult.endsWith('}')) {
+        try {
+          var decodedResponse = jsonDecode(verbResult);
+          throw AtExceptionUtils.get(decodedResponse['errorCode'],
+              decodedResponse['errorDescription']);
```

Comment:
This method (get(errorCode, errorDescription) isn't in the current AtExceptionUtils in at_tools ... can you tell me which PR it is in?

Reply:
It is in at_commons. Following is the PR: https://github.com/atsign-foundation/at_tools/pull/186/files